src/global.ts

```diff
@@ -1,10 +1,8 @@
-import { Prism } from './core/prism';
+import globalPrism, { type Prism } from './core/prism';
 
-const globalSymbol = Symbol.for('Prism global');
-
-// eslint-disable-next-line no-undef
-const namespace = globalThis as Partial<Record<typeof globalSymbol, Prism>>;
-const globalPrism = (namespace[globalSymbol] ??= new Prism());
+declare global {
+	const Prism: Prism | undefined;
+}
```

Comment:
```suggestion
declare global {
	interface globalThis {
		Prism: Prism | undefined;
	}
}
```

This seems to be a better approach. However, I still don't understand why TS complains in `config.ts`. I tried adding the index signature to the `globalThis` interface, but it didn't help. 🤷‍♂️

```ts
declare global {
	interface globalThis {
		Prism: Prism | undefined;
		[key: string]: unknown;
	}
}
```